docker/conda/environments/cuda11.8_dev.yml

```diff
@@ -82,24 +86,18 @@ dependencies:
     - python-confluent-kafka=1.9.2
     - python-graphviz
     - python=3.10
-    - pytorch=2.0.1
     - pytorch-cuda
+    - pytorch=2.0.1
     - rapidjson=1.1.0
     - scikit-build=0.17.1
     - scikit-learn=1.2.2
     - sphinx
     - sphinx_rtd_theme
     - sqlalchemy<2.0 # 2.0 is incompatible with pandas=1.3
     - sysroot_linux-64=2.17
-    - tritonclient=2.26 # Required by NvTabular, force the version, so we get protobufs compatible with 4.21
     - tqdm=4
+    - tritonclient=2.26 # Required by NvTabular, force the version, so we get protobufs compatible with 4.21
     - typing_utils=0.1
     - watchdog=2.1
+    - websockets
     - yapf=0.40.1
-    ####### Morpheus Pip Dependencies (keep sorted!) #######
-    - pip:
-        # Add additional dev dependencies here
-        - docutils
-        - pytest-kafka==0.6.0
```

Comment:
What happened to `pytest-kafka`?

Reply:
Good question. CI passes. Does it need to be installed in release? There's a script to install it in the devcontainer.

Reply:
`pytest-kafka` is needed for running the tests. We install it manually in the Dockerfile to ensure its setup for CI but this doesnt cover normal developer use. I think it should be left in the dev dependencies.